src/priorityQueue.js

```diff
@@ -0,0 +1,41 @@
+'use strict'
+
+class Node {
+  constructor(follows, data, priority, proceeds) {
+    this.after = follows
+    this.info = data
+    this.priority = priority
+    this.before = proceeds
+  }
+}
+
+class priorityQueue {
+  constructor() {
+    this.head = null
+    this.tail = null
+    this.count = 0
+  }
+}
+
+  enqueue(value, num) {// adds an element with priority (number) to the back of the queue.
+    let behind = this.tail
+    let inFront = null
+    const element = new Node(behind, value, num, inFront)
+    this.count === 0
+      ?(this.tail = element,
+        this.head = element)
+      :(while (num > behind.priority) {
```

Comment:
This is especially difficult to parse (and is syntactically incorrect - you're missing a parenthesis).